README.md

```diff
@@ -89,3 +89,28 @@ Index ID set to **ocp-product-docs-4_15**.
 
 These dictories and index ID can now be used to configure OpenShift
 Lightspeed.
+
+### Generating a RAG vector database using Postgres (pgvector)
```

Comment:
It would be better if this workflow followed steps from the original. So instead of making separate directory for source documents, it would use the already downloaded product docs for OCP

Reply:
Can we create a separate issue for this? Since I have not been dealing with OCP docs, it would take some time for implementing this for me  and probably someone from OLS team can modify scripts much faster and more elegantly than I.

Reply:
I think the change @jpodivin is requesting here is not that big (no need to implement anything). Please, correct me if I'm wrong Jirka. In the README above, this particular step downloads the OCP documentation under `ocp-product-docs-plaintext/4.15/`:

```
./scripts/get_ocp_plaintext_docs.sh 4.15
```

The request is about making sure that in the README we show how to generate the vector database from doc files stored under `ocp-product-docs-plaintext/4.15/`. 

I honestly think it would be nice as well:). It would keep things consistent.

Reply:
Well, let me spend some time on the weekend. I am just afraid of the size of of the OCP docs :-)

Reply:
The amount we are using with existing example isn't so bad. But if it does cause issue, I would recommend removing some of the documents before processing, keeping only a fraction. That way you can verify yourself that it works, and push changes that assume the entire batch is present.

Reply:
I have updated the sample and README to use OCP docs as input.